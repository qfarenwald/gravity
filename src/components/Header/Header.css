.Header-img {
  margin: 100px 100px 20px 100px;
  width: 60%;
}

.Header-h2 {
  color: white;
  font-weight: lighter;
  letter-spacing: 50px;
}
