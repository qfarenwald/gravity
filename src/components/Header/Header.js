import React from 'react';
import { Link } from 'react-router-dom';
import './Header.css';
import logo from '../../images/gravity_logo.svg';

export const Header = () => (
  <header className="Header">
    <Link to="/"><img className="Header-img" src={logo} alt="Gravity logo" /></Link>
    <h2 className="Header-h2">SHUTTLE SERVICE</h2>
  </header>
);
