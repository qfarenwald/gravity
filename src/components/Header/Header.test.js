import React from 'react';
import ReactDOM from 'react-dom';
import { Header } from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Header />)

    expect(wrapper).toMatchSnapshot()
  })
})
