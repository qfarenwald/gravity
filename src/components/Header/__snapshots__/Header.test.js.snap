// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<header
  className="Header"
>
  <Link
    to="/"
  >
    <img
      alt="Gravity logo"
      className="Header-img"
      src="gravity_logo.svg"
    />
  </Link>
  <h2
    className="Header-h2"
  >
    SHUTTLE SERVICE
  </h2>
</header>
`;
